### ISSUE TITLE
Peer review for proposal - Group 5

### ISSUE THREAD

**Issue by clarissdev** [Sadswefg/Data-Visualization---Group-Project-1#2]
Overall, your team has made a very clear and comprehensive proposal. It has provided me an overall idea of the project that you are going to do. The structure of dataset, the questions, and your plan are well defined and easy to understand. However, there are certain types of issues that I would like your group to consider as following:

- Please help to fix some typo (Volumn -> Volume, enconded  -> encoded, etc).
- The [Instruction for Project 1](https://vinuni.instructure.com/courses/1977/files/358331?module_item_id=86321) indicated that the proposal has to show the reason you choose the data. I think you should include this as a new section, indicating the importance of the dataset to the questions you are trying to answer.
- For the question 4 (What would be the solutions for reducing the trash dumping, especially for the non-biodegradable waste?), I think this is out of scope of the course, because the answers to the questions should be the insights drawn from the data. Some items in your plan (conduct educational campaigns, etc) might not relate as well.
- For the project plan, I think you should indicate more clearly about your plan on each specific questions. I would like to see more about how the dataset structure helps you answer the questions.

**Comment by tienvu95** [comment]
I agree with these notes, it showed that your group did read the proposal thoroughly and made the important suggestions for group F.

**Comment by Sadswefg** [comment]
Dear @clarissdev,

Thank you very much for taking the time to review our project proposal carefully and giving it really nice constructive feedback. We greatly appreciate your insights, which have helped us to refine our project further. We have carefully reviewed your comments and have made the following adjustments to our proposal:
1. Typos Corrected: We have addressed and corrected the typos you highlighted, such as "Volumn" to "Volume" and "enconded" to "encoded". We aim for our proposal to be as clear and error-free as possible.
2. Reason for choosing the data: your point about explaining why we chose our dataset was well taken. Yes, we definitely will put this into a new section to make our proposal much clearer about our goals to this project.
3. Refinement of Question 4: Upon reflection, we agree that Question 4, concerning solutions for reducing non-biodegradable waste, was beyond the scope of our course. We've decided to narrow our focus to two main questions that are directly related to insights we can draw from our data.
4. Clarification of Project Plan: We have revised our project plan to offer a more detailed explanation of how we intend to use the dataset structure to answer each specific question. This includes outlining our analytical approaches and how they relate to the data at hand, ensuring our objectives are clear and achievable.

Thank you again for your invaluable feedback. We look forward to possibly incorporating more of your suggestions as our project progresses.
